lib/sampler.py

```diff
@@ -142,11 +151,18 @@ def _sample(self):
             except ProcessLookupError:
                 self.unregister_process(pid)
             else:
-                self._sample_process(pid)
-
-    def _sample_process(self, pid):
-        # Your sampling code here.
-        pass
+                self._sample_process(pid, tasks_rss)
+        # save current overall RSS value if it is bigger than saved
```

Comment:
Nits:

1. Please, add an empty line to separate this block from the previous one.
2. Please, follow tarantool's commenting style, when a comment is started from a capital letter and ends with a period.

Reply:
Right, corrected all changes in all commits:
~/tarantool/test-run$ git show HEAD~0 | grep \#
```
    Closes #286
         # filled in execute() when a greenlet runs
         # cleanup only if test passed or if --force mode enabled
                 # run task and save the result to short_status
                 # check if the results file checksum set on fail and if
                 # the newly created results file is known by checksum
         # Prepare standalone subpath '<vardir>/statistics' for statistics files.
+        # Durations.
+        # Print to stdout durations for all failed tasks.
+        # Print to stdout durations for some number of most long tasks.
+        # Print durations statistics to '<vardir>/statistics/durations.log' file.
```
~/tarantool/test-run$ git show HEAD~1 | grep \#
```
    Closes #277
         # Called from a worker process (_run_worker()).
             # Unregister processes that're gone.
             # Assume that PIDs are rarely reused.
-        # Your sampling code here.
+        # Save current overall RSS value if it is bigger than saved.
+        # Count overall RSS per task.
 # The 'singleton' sampler instance: created in the main test-run
+        # Prepare standalone subpath '<vardir>/statistics' for statistics files.
+        # RSS.
+        # Print to stdout RSS statistics for all failed tasks.
+        # Print to stdout RSS statistics for some number of most it used tasks.
+        # Print RSS statistics to '<vardir>/statistics/rss.log' file.
```